ProcessOptimizer/optimizer/optimizer.py

```diff
@@ -467,11 +467,13 @@ def ask(self, n_points=None, strategy="stbr_fill"):
                 + "got %s" % strategy
             )
         
-        if strategy in ["stbr_fill", "stbr_full"] and self.get_constraints() is not None:
-            raise ValueError(
-                "Steinerberger (default setting) sampling can not be used with constraints,\
-                try using another strategy like 'opt.ask(n,strategy='cl_min')'"
-            )
+        if (n_points > self._n_initial_points 
```

Comment:
Could we throw a warning if we are within the initial points? Then we don't have "unexpected failures" once we run out of initial points.

Reply:
I've added a warning below the initial points.